### DISCUSSION TITLE
[7/5/2023] New Mojo Release!!

### DISCUSSION THREAD

**Discussion by goldiegadde** [modular/modular#431] [Announcements]
Hey everyone 👋 ,

We just updated Playground with a new Mojo🔥 release! To see what's new, [check out the changelog](https://docs.modular.com/mojo/changelog.html).

If you have an active session in the Mojo Playground, you'll need to restart it to get the updated build.
Thanks, 
Goldie

**Comment by drunkwcodes** [comment]
Pythonic update!
